docs/sources/enterprise/reporting.md

```diff
@@ -42,6 +42,16 @@ Currently only Organization Admins can create reports.
 
 {{< docs-imagebox img="/img/docs/enterprise/reports_create_new.png" max-width="500px" class="docs-image--no-shadow" >}}
 
+## Send test mail
+A test mail can be sent by pressing the Send test mail button. This will bring up a modal window where an email address or
+a list of email addresses can be entered. If the mail should use the mail addresses from the report instead,
+check the Use emails from report checkbox. 
+
+*Note that it is the last saved version of the report that will be sent and changes made to the report after saving will not be
```

Comment:
Rewrite this to be the intro to the numbered steps above, placed under the heading. 

The last saved version of the report will be sent to selected emails. You can use this to verify emails are working and to make sure the report generates and is displayed as you expect.